decentraland/screens/EthereumSignUpScreen.js

```diff
@@ -3,25 +3,41 @@ import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import { setAccount } from "../redux/actions";
 import EthereumSignUp from "@presentational/EthereumSignUp";
-import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+import {
+  approveManaSpending,
+  showInfo,
+  showError,
+  fundAccount,
+} from "./helpers";
+
+import { Account } from "tasit-sdk";
 
 export class EthereumSignUpScreen extends React.Component {
-  // Note: As same as Account.create(), this functions isn't running as async.
-  // Timeout Between button click and screen change (afterSignUp()) is abount 5 secs.
-  // See more: https://github.com/tasitlabs/tasit/issues/42
-  _createAccount = async () => {
-    const { setAccount } = this.props;
-    // Note: A real app wouldn't be using a preset private key and hardcoding it!
-    // We're only doing this temporarily while using a hardcoded account with ETH and tokens
-    const accountPrivateKey =
-      "0x4f09311114f0ff4dfad0edaa932a3e01a4ee9f34da2cbd087aa0e6ffcb9eb322";
-    const account = createFromPrivateKey(accountPrivateKey);
-    setAccount(account);
+  // TODO: Use Gnosis Safe
+  _onboarding = async () => {
+    try {
+      const { setAccount } = this.props;
+
+      // Note: The timeout for account creation is about ~10 secs.
+      // See more: https://github.com/tasitlabs/tasit/issues/42
```

Comment:
How do we handle this from a UX perspective right now? We should show that an account is being created.

Reply:
OK, I'm using the `YellowBox` development warnings for that right now. I agree with you and I think that the next UX improvement is to use properly components to show loading/error/success messages to the user.

**Note**: About the creation process, I'm trying to do that:

```javascript
    showInfo(`Creating and funding account...`);

    // Should run async but isn't when calling Account.create() or createFromPrivateKey()
    this._onboarding();
```

Even `_onboarding();` is an async function, the info message above is showing after the `Account.create()` process.

Reply:
Yeah, using YellowBox for now was a great call.

Got it, that works for the account creation process loading state.

Wait, so you are you saying that the info message doesn't ever go away? Or just that it takes until the end of the account creation process to go away?

Reply:
1) The YellowBox warn don't go away.
2) It took until the end of the account creation process to be shown.

Reply:
Oh interesting, that's too bad. Maybe if we want a few ticks before starting the account creation process after trying to show the yellow box it would show up beforehand. I remember this relates to an older issue we had with account creation. Maybe if we use local state for the component or global state to track if an account has been created we can make sure we hide the warning explicitly later?

Reply:
1) About account creation message, I'll try your suggestion to round that, thanks.
2) About warn/messages I've created that issue: https://github.com/tasitlabs/tasit/issues/159

The issue is this one: https://github.com/tasitlabs/tasit/issues/42#issuecomment-462534793

Reply:
Ah yep, that was the issue. Okay sounds good, thanks.